marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -88,60 +110,104 @@ const Browse: React.FC<Props> = ({location}) => {
         const parsedModelData = entityFromJSON(response.data);
         let entityArray = [...entityFromJSON(response.data).map(entity => entity.info.title)];
         let parsedEntityDef = entityParser(parsedModelData);
-        setEntites(entityArray);
+        setEntities(entityArray);
         setEntityDefArray(parsedEntityDef);
         setEntityDefinitionsArray(parsedEntityDef);
         setEntitiesData(response.data);
+        return entityArray;
       }
     } catch (error) {
       handleError(error);
     }
   };
 
+  const fetchResults = async (database: string, entitiesList) => {
+    const {query, selectedFacets, selectedTableProperties, start, pageLength, sortOrder} = searchOptions;
+    const {data} = await axios({
```

Comment:
Let's move the api declarations under src/api folder and just use the references here, if possible.